src/digitalIdentity-backend/src/main/java/didentity/amos/digitalIdentity/controller/CredentialDefinitionController.java

```diff
@@ -28,13 +28,14 @@ public class CredentialDefinitionController {
             @RequestParam(required = false) String authorization,
             @RequestParam String alias,
             @RequestParam String comment,
-            @RequestParam String schemaId) {
+            @RequestParam String schemaId,
+            @RequestParam String revocable) {
 
         if (authenticationService.authentication(authorization) == false) {
             return authenticationService.getError();
         }
 
```

Comment:
You want to check, whether revocable is valid ("", or "true" or "false")

Reply:
```suggestion
        if (revocable!= null && !(revocable.equals("false") || revocable.equals("true"))) {
            return ResponseEntity.status(400).body("Bad Request. If present, revocable shall be 'true' or 'false'.");
        }
```

Reply:
@Idontker Das ist nicht nötig, das Lissi das schon für uns übernimmt. Wenn man mit dem Postman den Endpunkt ausprobierst, dann verhält sich so das für folgenden Inhalt im Parameter revocable:

### Parameter revocable ist "true"
Erstellt eine neue Credential-definition und gibt diese als Json zurück
```json
{
    "id": "GCevMyEWCa5Fd58gfzkASy:3:CL:8768:testJannik3",
    "alias": "testJannik3",
    "comment": "test",
    "imageUri": "http://onboardingad.ddns.net/ctrl/images/download/GCevMyEWCa5Fd58gfzkASy:3:CL:8768:testJannik3",
    "schemaId": "GCevMyEWCa5Fd58gfzkASy:2:Mitarbeiter Ausweis:1.00",
    "active": true,
    "revocable": true,
    "version": "1.0",
    "imported": false
}
```

### Parameter revocable ist "false"
Erstellt eine neue Credential-definition und gibt diese als Json zurück
```json
{
    "id": "GCevMyEWCa5Fd58gfzkASy:3:CL:8768:testJannik4",
    "alias": "testJannik4",
    "comment": "test",
    "imageUri": "http://onboardingad.ddns.net/ctrl/images/download/GCevMyEWCa5Fd58gfzkASy:3:CL:8768:testJannik4",
    "schemaId": "GCevMyEWCa5Fd58gfzkASy:2:Mitarbeiter Ausweis:1.00",
    "active": true,
    "revocable": false,
    "version": "1.0",
    "imported": false
}
```

### Parameter revocable ist anders als "true" und "false", z. B. "test"
Gibt als response eine Fehlermeldung zurück.
```
Could not create a new credential.
```